Logger/LogFactory.cs

```diff
@@ -1,10 +1,17 @@
-﻿namespace Logger;
+namespace Logger;
 
 public class LogFactory
 {
+    private string? _filePath;
+
     public BaseLogger CreateLogger(string className)
     {
-        
-        return null;
+        if (_filePath is null) { return null!; }
```

Comment:
```suggestion
        if (_filePath is null) { return null; }
```
avoid using `!` whenever possible, if you find yourself using this, ask yourself if there is a better solution. In this case, the better solution is to change your method signature to return a nullable `BaseLogger` on line 5. 

Like this:
`public BaseLogger? CreateLogger(string className)`